src/util/worker/mod.rs

```diff
@@ -91,18 +91,21 @@ pub struct Scheduler<T> {
     name: Arc<String>,
     counter: Arc<AtomicUsize>,
     sender: Sender<Option<T>>,
+    busy_limit: usize,
 }
 
 impl<T: Display> Scheduler<T> {
     fn new<S: Into<String>>(
         name: S,
         counter: AtomicUsize,
         sender: Sender<Option<T>>,
+        busy_limit: usize,
```

Comment:
What's the purpose of `busy_limit`?

Reply:
It's for `Scheduler` can adjust `is_busy` behavior: if number of pending tasks in channel exceeds `busy_limit`, `Scheduler::is_busy()` will return true.

Reply:
However, it's expected that the messages in channel should not exceed `capacity`. I think it's different from `is_busy`.

Reply:
pending_limit ?

Reply:
@BusyJay The channel is unbounded, so I think we need a `limit` to compare with `counter`.

Reply:
Why not use bounded channel?

Reply:
So I think we don't nee `is_busy` and `counter` any more?

Reply:
Why? I believe `is_busy` is a used function. `counter` is also needed for metrics.